### DISCUSSION TITLE
Input file

### DISCUSSION THREAD

**Discussion by YanghaoCui** [CenterForTheBuiltEnvironment/ABCWeb#58] [Q&A]
Thanks for developing this useful toolbox! I am connecting the API of ABC_toolbox for running a trial since the online live tool was unable to open (It may be fixed recently).  My question is about the input file. I want to know more about the setting of **Comfort Model** since I want to use skin temperatures to run the simulation directly. Could you provide an example or something. Thanks.

**Comment by AkihisaNomoto** [comment]
Hi @YanghaoCui, Thank you for asking and sorry for the problem of the online tool. It is still under development but you should be able to run here (https://abcweb-384255928646.us-west1.run.app/) for now. In the near future, we will probably change the domain so please note that the URL of the current online tool will be changed again. We have an option for users to directly input skin temperature to predict comfort but we are not ready to release that feature. So stay tuned. I will definitely get back to you once we perfectly implement that feature.

**Comment by YanghaoCui** [comment]
Hi Akihisa,

Thanks for sharing the updated link. I will follow up on the updates of ABC Tool.


Best regards / Med venlig hilsen,


[cid:165f79a6-73e0-40f3-a342-b11a0f600fbb]

Yanghao Cui

PhD student

International Centre for Indoor Environment and Energy (ICIEE)

DTU SUSTAIN, Technical University of Denmark



Mob. +45 52 67 82 75

***@***.***

Koppels Allé

Building 402, Room 227

2800 Kgs. Lyngby

www.dtu.dk/english<https://www.dtu.dk/english>


________________________________
